### PR TITLE
quarterly top up, finalise stable farm rebalance and sweep vault dust 

### PR DESCRIPTION
**Pull request by jorijnsmit**
- quarterly top up (#502)
- deposit and stake bpt_3pool (last action in #413)
- also a general clean up of lots of dust in the vault, which is sent to trops

the preview doesnt like `safe.balancer.deposit_and_stake` hence `skip_preview=True`

### CODE REVIEW

File: scripts/issue/502/top_ups_for_q3.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    # stake ~$7k worth of dust bpt
+    safe.init_balancer()
+    safe.balancer.stake_all(bpt_badgerwbtc)
+
+    # balancer 3pool deposit & stake;
+    # https://github.com/Badger-Finance/badger-multisig/issues/413
+    underlyings = [usdt, usdc, dai]
+    amounts = [int(102_457e6), int(102_457e6), int(102_457e18)]
+    safe.balancer.deposit_and_stake(underlyings, amounts)
+    safe.balancer.stake_all(bpt_3pool) * .995
```
Comment (petrovska-petro) [inline scripts/issue/502/top_ups_for_q3.py:59]:
```suggestion
    safe.balancer.stake_all(bpt_3pool, dusty=.995)
```

File: scripts/issue/502/top_ups_for_q3.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        badger, usdc, weth, bal, usdt, dai, eurs, crv, cvx, fei, angle,
+        threepool, ibbtc_lp, slp, bpt_badgerwbtc, bpt_3pool, bvecvx
+    ])
+    trops.take_snapshot([
+        badger, usdc, weth, bal, usdt, dai, eurs, crv, cvx, fei, angle,
+        threepool, ibbtc_lp, slp, bpt_badgerwbtc, bpt_3pool, bvecvx
+    ])
+
+    # badger for emissions, rembadger, bribes and contribs q3
+    badger.transfer(r.drippers.tree_2022_q2, 260_000e18)
```
Comment (petrovska-petro) [inline scripts/issue/502/top_ups_for_q3.py:37]: maybe for the drippers already use dummy placers as `r.drippers.tree_2022_q3` & `r.drippers.rembadger_2022_q3` even if using same address as q2, so if it gets merged before deploying the q3 ones, then it just matter of updating the address book or worth deploying them already and include them in this pr

Comment (jorijnsmit) [reply scripts/issue/502/top_ups_for_q3.py]: pushed in eb1249f

**Review by petrovska-petro — APPROVED** [review]
LGTM!

For other reviewers, figures deducted from [L40](https://github.com/Badger-Finance/badger-multisig/blob/issue/502/scripts/issue/502/top_ups_for_q3.py#L40) & [L43](https://github.com/Badger-Finance/badger-multisig/blob/issue/502/scripts/issue/502/top_ups_for_q3.py#L43) are coming from the fact of funds in this [msig](https://github.com/Badger-Finance/badger-multisig/blob/main/helpers/addresses.py#L44) to be transferred to the *payments_multisig* (mainly all the badger funds).

**Review by jalbrekt85 — APPROVED** [review]
lgtm. agree that it is better to just finish the balancer stable deposit at this point.

**Pull request merged**